frappe/public/js/frappe/model/create_new.js

```diff
@@ -83,7 +83,8 @@ $.extend(frappe.model, {
 		var cnt = frappe.model.new_name_count;
 		if (!cnt[doctype]) cnt[doctype] = 0;
 		cnt[doctype]++;
```

Comment:
Remove `cnt` if it's not being used? or is it used anywhere else?

Reply:
we are still using the number, {doctype}-${cnt[doctype]}${hash}